portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -113,6 +258,67 @@ definition {
 		}
 	}
 
+	@description = "LPS-122873 When all the web content is unassigned from a Vocab1's category, the relative ordering of the rest of the categories remain the same"
+	@priority = "5"
+	test Category2OfVocab1NotSelected {
+		property portal.acceptance = "true";
```

Comment:
Change the title of this test, is not easy to know what it does. Dont refer it to concrete names, it has to say something related with the order or something like that

Reply:
This name remains the same, please modify.